### PR TITLE
Provide descriptions for Windows installer components, fixes #903

### LINKED ISSUE #903: Provide a description for the components of the Windows installer
As already mentioned at the TYPO3 DDEV Slack channel it would be nice to have a description for the components and their use case.

I will provide a initial PR for that issue a.s.a.p.

### PR DESCRIPTION
**Pull request by gilbertsoft**
## The Problem/Issue/Bug:
#903 

## How this PR Solves The Problem:
Provides a description for each component of the Windows installer

## Manual Testing Instructions:
Start the installer and move the mouse over the components to see the new descriptions

## Release/Deployment notes:
Works with NSIS for POSIX only as before. To use with the Windows version slashes in the file paths has to be changed to backslashes

### CODE REVIEW

File: winpkg/ddev.nsi [hunk 1]
```diff
@@ -15,9 +15,9 @@ CRCCheck On
 !include LogicLib.nsh
 
 ; The name of the installer
-Name "ddev ${VERSION}"
+Name "ddev ${MUI_VERSION}"
```
Comment (gilbertsoft) [inline winpkg/ddev.nsi]: This is not necessary but IMO makes sense to use only the previously defined constant

File: winpkg/ddev.nsi [hunk 2]
```diff
@@ -15,9 +15,9 @@ CRCCheck On
...
 
-OutFile "../bin/windows/windows_amd64/ddev_windows_installer.${VERSION}.exe"
+OutFile "../bin/windows/windows_amd64/ddev_windows_installer.${MUI_VERSION}.exe"
```
Comment (gilbertsoft) [inline winpkg/ddev.nsi]: same as above

File: winpkg/ddev.nsi [hunk 3]
```diff
@@ -88,25 +90,41 @@ Section "ddev (github.com/drud/ddev)"
...
+Section "Start Menu Shortcuts" SecStartMenu
   CreateDirectory "$SMPROGRAMS\ddev"
   CreateShortcut "$SMPROGRAMS\ddev\Uninstall.lnk" "$INSTDIR\ddev_uninstall.exe" "" "$INSTDIR\ddev_uninstall.exe" 0
 SectionEnd
 
 ;--------------------------------
+;Descriptions
+
+  ;Language strings
+  LangString DESC_SecDDEV ${LANG_ENGLISH} "Install DDEV-local (required)."
```
Comment (gilbertsoft) [inline winpkg/ddev.nsi]: please check this descriptions as mentioned at Slack

**Comment by rfay** [comment]
I took the liberty of pushing the suggested changes into your fork/branch.

**Comment by rfay** [comment]
I downloaded the (successful) circle build artifacts and tested on Windows, and this looks good to go. I don't know why circle results are not being shown. I'd like to get this in in spite of the ddev-containers-windows failing, and I'll do a PR to fix that (hopefully).

**Comment by rfay** [comment]
This was a [clean green build](https://buildkite.com/drud/ddev-containers-windows/builds/33) despite the fact that it shows ddev-containers-windows failing.

**Comment by rfay** [comment]
Thanks so much for this @gilbertsoft ! We look forward to more contributions in the future.

**Pull request merged**